tests/attr/test_input_x_gradient.py

```diff
@@ -1,55 +1,58 @@
 #!/usr/bin/env python3
-
+from typing import Any
 import torch
+from torch import Tensor
+from torch.nn import Module
 
 from captum.attr._core.input_x_gradient import InputXGradient
 from captum.attr._core.noise_tunnel import NoiseTunnel
+from captum.attr._utils.typing import TensorOrTupleOfTensors
 
 from .helpers.classification_models import SoftmaxModel
 from .helpers.utils import assertArraysAlmostEqual, BaseTest
 from .test_saliency import _get_basic_config, _get_multiargs_basic_config
 
 
 class Test(BaseTest):
-    def test_input_x_gradient_test_basic_vanilla(self):
+    def test_input_x_gradient_test_basic_vanilla(self) -> None:
         self._input_x_gradient_base_assert(*_get_basic_config())
 
-    def test_input_x_gradient_test_basic_smoothgrad(self):
+    def test_input_x_gradient_test_basic_smoothgrad(self) -> None:
         self._input_x_gradient_base_assert(*_get_basic_config(), nt_type="smoothgrad")
 
-    def test_input_x_gradient_test_basic_vargrad(self):
+    def test_input_x_gradient_test_basic_vargrad(self) -> None:
         self._input_x_gradient_base_assert(*_get_basic_config(), nt_type="vargrad")
 
-    def test_saliency_test_basic_multi_variable_vanilla(self):
+    def test_saliency_test_basic_multi_variable_vanilla(self) -> None:
         self._input_x_gradient_base_assert(*_get_multiargs_basic_config())
 
-    def test_saliency_test_basic_multi_variable_smoothgrad(self):
+    def test_saliency_test_basic_multi_variable_smoothgrad(self) -> None:
         self._input_x_gradient_base_assert(
             *_get_multiargs_basic_config(), nt_type="smoothgrad"
         )
 
-    def test_saliency_test_basic_multi_vargrad(self):
+    def test_saliency_test_basic_multi_vargrad(self) -> None:
         self._input_x_gradient_base_assert(
             *_get_multiargs_basic_config(), nt_type="vargrad"
         )
 
-    def test_input_x_gradient_classification_vanilla(self):
+    def test_input_x_gradient_classification_vanilla(self) -> None:
         self._input_x_gradient_classification_assert()
 
-    def test_input_x_gradient_classification_smoothgrad(self):
+    def test_input_x_gradient_classification_smoothgrad(self) -> None:
         self._input_x_gradient_classification_assert(nt_type="smoothgrad")
 
-    def test_input_x_gradient_classification_vargrad(self):
+    def test_input_x_gradient_classification_vargrad(self) -> None:
         self._input_x_gradient_classification_assert(nt_type="vargrad")
 
     def _input_x_gradient_base_assert(
         self,
-        model,
-        inputs,
-        expected_grads,
-        additional_forward_args=None,
-        nt_type="vanilla",
-    ):
+        model: Module,
+        inputs: TensorOrTupleOfTensors,
+        expected_grads: Tensor,
```

Comment:
This should also be TensorOrTupleOfTensors

Reply:
Changing `expected_grads` from `Tensor` to `TensorOrTupleOfTensors` results in the following error in 3 places:
`tests/attr/test_input_x_gradient.py:35: error: Value of type variable "TensorOrTupleOfTensors" of "_input_x_gradient_base_assert" of "Test" cannot be "object"`